woocommerce/woocommerce-android
# [Shipping Labels Revamp] Add Custom package creation support

Summary
==========
Introduce the endpoint, models, and networking code to fix issue #12283 and properly submit a Custom package to the store.

How to Test
==========
1. Go to the new Shipping Labels form
2. Hit the `Select Package` button
3. In the Custom package tab, fill the package form. Ensure the `Save this as a new package template` option is checked.
4. Hit the `Add Package` button
5. The UI is not currently exiting the Package selection UI, so exit from the Package selection UI manually and hit the `Select Package` again. 
6. Go to the Saved Package tab and verify that the new Package is listed there.

Update release notes:


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
